packages/command-utils/package.json

```diff
@@ -29,6 +29,6 @@
   },
   "sideEffects": false,
   "devDependencies": {
-    "@types/which": "^2.0.0"
+    "@types/which": "^2"
```

Comment:
I've made these type of changes so `yarn upgrade-interactive` shows minor version changes. Also this is a more precise notation of what dependency we require, e.g. in most cases we are OK with someLibrary version 2.x, not just someLibrary version greater than 2.3.9, less than 3